myhpi/core/templates/core/information_page.html

```diff
@@ -17,9 +18,22 @@ <h1 class="page-title">
         <div class="side-panel-container">
             <aside class="side-panel border-accent">
                 <h1 class="toc-title">{% translate "Last edited" %}</h1>
-                {{ page.last_published_at|date:"d.m.Y, H:i" }}
+                {% get_current_timezone as TIMEZONE %}
+                <time id="last-published" datetime="{{ page.last_published_at|date:"c" }}" title="{{ TIMEZONE }}">
+                    <span id="last-published-date">
+                        {{ page.last_published_at|localtime|date:"SHORT_DATE_FORMAT" }}
```

Comment:
no need for localtime, tz_aware times (which those should be) will automatically be converted by django